packages/app/src/core/repl.ts

```diff
@@ -299,7 +300,7 @@ const stripTrailer = (str: string) => str && str.replace(/\s+.*$/, '')
 
 /** turn --foo into foo and -f into f */
 const unflag = (opt: string) => opt && stripTrailer(opt.replace(/^[-]+/, ''))
-
+const key = 'openwhisk.export'
```

Comment:
this key is shared between repl.ts and export.ts;

1) it is misnamed; the "openwhisk" part is not relevant
2) should be shared